### ISSUE TITLE
rotate method is missing from index.d.ts

### ISSUE THREAD

**Issue by MattBroe** [josdejong/mathjs#2385]
My code (relevant section is rotateXY() method):

`

import math from "mathjs"

export abstract class GeometryUtils {

    public static getDirection(vector: math.MathArray): math.MathArray {
        let norm = math.norm(vector);
        if (norm === 0) {
            return vector;
        }

        return math.divide(vector, norm) as math.MathArray;
    }

    public static rotateXY(vector: math.MathArray): math.MathArray {
        return math.rotate(vector, [0, 0, 1]);
    }
}

`

My tsc error: 

`

tsc
ts/geometry_utils.ts:15:21 - error TS2339: Property 'rotate' does not exist on type 'MathJsStatic'.

15         return math.rotate(vector, [0, 0, 1]);
                       ~~~~~~


Found 1 error.

`

Looking at index.d.ts in the repo confirms. mathjs version is 10.0.2. Has this been deprecated?

**Comment by josdejong** [comment]
I think the definition for `rotate` is simply missing. 

Anyone able to add this missing definition to `index.d.ts`?

**Comment by DIVYA-19** [comment]
@josdejong I made a pull request for this issue. could you have a look on that, please

**Comment by josdejong** [comment]
Fxied via #2419, thanks @DIVYA-19

**Comment by josdejong** [comment]
Fix is published in `v10.2.0` now
